libbeat/docs/outputconfig.asciidoc

```diff
@@ -739,21 +739,39 @@ use in Logstash for indexing and filtering:
 for more about the `@metadata` field.
 <2> The default is {beat_default_index_prefix}. To change this value, set the
 <<logstash-index,`index`>> option in the {beatname_uc} config file.
-<3> The beats current version.
-<4> The value of `type` is currently hardcoded to `doc`. It was used by previous
-Logstash configs to set the type of the document in Elasticsearch.
-
+<3> The current version of {beatname_uc}.
+endif::[]
 
-WARNING: The `@metadata.type` field, added by the Logstash output, is
-deprecated, hardcoded to `doc`, and will be removed in {beatname_uc} 7.0.
```

Comment:
This has been deprecated for a very long time and was not removed in `7.0` like the note says. I opted to remove this note and the `type` from the metadata. If you prefer to keep it around, let me know and I'll change this.